helm_deploy/use-of-force/values.yaml

```diff
@@ -11,5 +11,5 @@ generic-data-analytics-extractor:
   enabled: true
 
 ingress:
-  annotations:
-    external-dns.alpha.kubernetes.io/aws-weight: "100"
+  v1_2_enabled: true
+  v0_47_enabled: true
```

Comment:
🤞 works like the generic-service implementation, where we should see the traffic slowly move over.